src/components/ListArea/ListArea.js

```diff
@@ -0,0 +1,35 @@
+import './ListArea.css';
+import React, {useContext, useEffect, useState} from 'react';
+import {ListContext} from '../../context'
+import _ from 'lodash'
+
+function ListArea({title, listIndex=-1, readonly=false}){
+    
+    const [value, setValue] = useState()
+    const {listA, listB, setListByIndex, getListByIndex} = useContext(ListContext)
+
+    useEffect(() => {
+        if(readonly){
+            const displayList = _.join(getListByIndex(listIndex), '\n')
+            setValue(displayList)
+        }
+    }, [listA, listB, getListByIndex, listIndex, readonly])
+
+    const onTextInput = (event) => {
+        setValue(event.target.value);
+        console.log(event.target.value)
+        const valueList = _.split(event.target.value, '\n')
+        console.log(valueList)
+        setListByIndex(valueList,listIndex)
+    }
+
+    const isReadonly = readonly === "true"? true: false
```

Comment:
you don't need this ternary if.

Reply:
For some reason the prop is passed in as a string and the compiler yells at me

Reply:
There's a comment on that too. What I meant here is that everything after the ? is unneeded because the expression will return a boolean value